tests/configuration_spec.lua

```diff
@@ -54,6 +54,43 @@ describe("configuration", function()
         })
     end)
 
+    it("can define and use multi-byte mappings", function()
+        require("nvim-surround").setup({
+            surrounds = {
+                -- multi-byte quote
+                ["’"] = {
+                    add = { "’", "’" },
+                    delete = "^(’)().-(’)()$",
+                },
+            },
+        })
+        set_lines({ "hey! hello world" })
+        set_curpos({ 1, 7 })
+        vim.cmd("normal ysiw’")
+        check_lines({ "hey! ’hello’ world" })
+        vim.cmd("normal ds’")
+        check_lines({ "hey! hello world" })
+    end)
+
+    it("can define and use 'interpreted' multi-byte mappings", function()
+        require("nvim-surround").setup({
+            surrounds = {
+                -- interpreted multi-byte
+                ["<M-]>"] = {
+                    add = { "[[", "]]" },
```

Comment:
I think you forgot to specify the `find` key here, since the default will incorrectly search for `<M-]>.-<M-]>`; try
```lua
                    add = { "[[", "]]" },
                    find = "%[%b[]%]",
                    delete = "^(%[%[)().-(%]%])()$",
```

Reply:
Yay that works \o/